src/Quill.vue

```diff
@@ -1,139 +1,119 @@
-                            <span data-value="center" class="ql-picker-item"></span>
-                            <span data-value="right" class="ql-picker-item"></span>
-                            <span data-value="justify" class="ql-picker-item"></span>
-                        </span>
-                    </span>
-                </div>
-            </slot>
-        </div>
-        <div class="ui attached segment" v-el:quill @click.prevent="focusEditor"></div>
-    </div>
+  <div class="ui attached segment"
+       @click.prevent="focusEditor"
+       v-on:focus-editor="focusEditor"
+       v-on:set-html="setHTML"
+       v-on:set-content="setContents">
+  </div>
 </template>
 
 <script>
     const Quill = require('quill')
     export default {
         props: {
-            content: {},
-
-            author: {},
-
-            formats: {
-                type: Array,
-                default() {
-                    return []
-                },
-            },
-
-            keyBindings: {
-                type: Array,
-                default() {
-                    return []
-                },
-            },
-
-            output: {
-                default : 'delta'
-            },
-
-            keyup: {
-                default : null
-            },
-
-            config: {
-                default() {
-                    return {
-                        modules: { 'link-tooltip': true },
-                        theme: 'snow',
-                    }
-                },
-            },
+          content: {},
+          author: {},
+          formats: {
+            type: Array,
+            default: () => [],
+          },
+          keyBindings: {
+            type: Array,
+            default: () => [],
+          },
+          output: { default : 'delta', },
+          keyup: { default : null, },
+          config: {
+            type: Object,
+            default: () => { return {} },
+          },
         },
 
         data() {
-            return {
-                editor: {},
-            }
+          return {
+            editor: {},
+            defaultConfig: {
+              modules: {
+                toolbar: [
+                  [{ 'font': [] }],
+                  [{ 'header': [1, 2, 3, 4, 5, 6, false] }],
+                  ['bold', 'italic', 'underline', 'strike'],
+                  ['blockquote', 'code-block'],
+                  [
+                    { 'list': 'ordered' }, { 'list': 'bullet' }
+                  ],
+                  [
+                    { 'script': 'sub' }, { 'script': 'super' }
+                  ],
+                  [
+                    { 'indent': '-1' }, { 'indent': '+1' }
+                  ],
+                  [
+                    { 'color': [] }, { 'background': [] }
+                  ],
+                  [{ 'align': [] }],
+                  ['clean'],
+                ],
+              },
+              theme: 'snow',
+            },
+          },
```

Comment:
This line should end with a semicolon or nothing. The comma was preventing the build step in my project.